### PR TITLE
Updated Readme and config file to support Cloud exteded support deplo…

### PR DESCRIPTION
**Pull request by BalajiUmmadisetti**
…yment

### CODE REVIEW

**Comment by InDieTasten** [comment]
Nice pull requ...

**Comment by InDieTasten** [comment]
...est 😄

**Review by samarthasthana — CHANGES_REQUESTED** [review]
Please review the comments

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Cloud.cscfg [hunk 1]
```diff
@@ -3,7 +3,6 @@
   <Role name="CRWorkerRole">
     <Instances count="4" />
     <ConfigurationSettings> 
-      <Setting name="Microsoft.WindowsAzure.Plugins.Diagnostics.ConnectionString" value="DefaultEndpointsProtocol=https;AccountName=$storage$;AccountKey=$storageKey$" />
```
[the remark this answers is not in the document]
Comment (BalajiUmmadisetti) [reply Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Cloud.cscfg:6]: as per https://learn.microsoft.com/en-us/azure/cloud-services-extended-support/deploy-prerequisite. Remove old diagnostics settings for each role in the Service Configuration (.cscfg) file.

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Cloud.cscfg [hunk 2]
```diff
@@ -3,7 +3,6 @@
...
       <Setting name="ServiceDnsName" value="%ServiceDns%" />
```
[the remark this answers is not in the document]
Comment (BalajiUmmadisetti) [reply Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Cloud.cscfg:6]: done

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Cloud.cscfg (hunk 1 above)
[the remark this answers is not in the document]
Comment (BalajiUmmadisetti) [reply Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Cloud.cscfg:6]: added again, if not required please let me know.

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Cloud.cscfg [hunk 3]
```diff
@@ -16,8 +15,14 @@
     </Certificates> 
   </Role>
   <NetworkConfiguration>
+	  <!-- Name of VirtualNetworSite.-->
+	<VirtualNetworkSite name="PolicyRecordingBot" />
     <AddressAssignments>
       <InstanceAddress roleName="CRWorkerRole">
+		  <!-- Name of Subnets.-->
+		  <Subnets>
```
[the remark this answers is not in the document]
Comment (BalajiUmmadisetti) [reply Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Cloud.cscfg:23]: included

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Local.cscfg [hunk 4]
```diff
@@ -15,13 +14,19 @@
...
+		<!-- Name of VirtualNetworSite.-->
+		<VirtualNetworkSite name="PolicyRecordingBot" />
+		<AddressAssignments>
+			<InstanceAddress roleName="CRWorkerRole">
+				<!-- Name of Subnets.-->
+				<Subnets>
+					<Subnet name="default"/>
+				</Subnets>
+				<PublicIPs>
+					<PublicIP name="instancePublicIP" domainNameLabel="pip" />
```
[the remark this answers is not in the document]
Comment (BalajiUmmadisetti) [reply Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Local.cscfg:27]: mentioned in the readme steps

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceDefinition.csdef [hunk 5]
```diff
@@ -21,7 +21,6 @@
       </Task>
     </Startup>
     <ConfigurationSettings>
-      <Setting name="Microsoft.WindowsAzure.Plugins.Diagnostics.ConnectionString" />
```
[the remark this answers is not in the document]
Comment (BalajiUmmadisetti) [reply Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceDefinition.csdef:24]: https://learn.microsoft.com/en-us/azure/cloud-services-extended-support/deploy-prerequisite. Remove old diagnostics settings for each role in the Service Configuration (.cscfg) file.
<Setting name="Microsoft.WindowsAzure.Plugins.Diagnostics.ConnectionString" value="UseDevelopmentStorage=true" />

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Cloud.cscfg (hunk 1 above)
[the remark this answers is not in the document]
Comment (BalajiUmmadisetti) [reply Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Cloud.cscfg:6]: as per not required https://learn.microsoft.com/en-us/azure/cloud-services-extended-support/deploy-prerequisite. Remove old diagnostics settings for each role in the Service Configuration (.cscfg) file.

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Local.cscfg [hunk 6]
```diff
@@ -3,25 +3,38 @@
...
       <Setting name="ServiceDnsName" value="%ServiceDns%" />
-      <!-- xyz.cloudapp.net-->
-      <Setting name="ServiceCNAME" value="%CName%" />
-      <!-- CNAME pointing to the .cloudapp.net if available.-->
+		<!-- xyz.cloudapp.net-->
+		<!-- Service DNS Name: 
+         This setting defines the DNS name for the service. 
+         It should be the fully qualified domain name (FQDN) used to access the service, like 'myservice.cloudapp.net'. 
+         Replace %ServiceDns% with your service's DNS name. -->
+		<Setting name="ServiceCNAME" value="%CName%" />
```
Comment (samarthasthana) [inline Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Local.cscfg:12]: Fix Indentation

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Local.cscfg [hunk 7]
```diff
@@ -3,25 +3,38 @@
...
-      <InstanceAddress roleName="CRWorkerRole">
-        <PublicIPs>
-          <PublicIP name="instancePublicIP" domainNameLabel="pip" />
-        </PublicIPs>
-      </InstanceAddress>
-    </AddressAssignments>
-  </NetworkConfiguration>
+	<NetworkConfiguration>
+		<!-- Name of VirtualNetworSite.-->
+		<VirtualNetworkSite name="PolicyRecordingBot" />
```
Comment (samarthasthana) [inline Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Local.cscfg:27]: Fix indentation for this section to be like the above section, use spaces not tabs.

File: Samples/configure_cloud.ps1 [hunk 8]
```diff
@@ -18,9 +18,6 @@ Enter your Service DNS name (ex: contoso.cloudapp.net).
 .PARAMETER CName
 Enter your Service CName (ex: contoso.net).
 
-.PARAMETER CertThumbprint
```
Comment (samarthasthana) [inline Samples/configure_cloud.ps1:21]: Is the `configure_cloud.ps1` still valid? Are we using it in the steps above to configure the files? If not we can clean it up?

Comment (samarthasthana) [reply Samples/configure_cloud.ps1:21]: Sync'd and will keep the change.

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Local.cscfg (hunk 6 above)
Comment (BalajiUmmadisetti) [reply Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Local.cscfg]: done

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Local.cscfg (hunk 7 above)
Comment (BalajiUmmadisetti) [reply Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/ServiceConfiguration.Local.cscfg]: done

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md [hunk 9]
```diff
@@ -57,25 +59,138 @@ To verify your policy was assigned correctly:
...
-2. Set up SSL certificate and upload to the cloud service
-    1. Create a wildcard certificate for your service. This certificate should not be a self-signed certificate. For instance, if your bot is hosted at `bot.contoso.com`, create the certificate for `*.contoso.com`.
-    2. Upload the certificate to the cloud service.
-    3. Copy the thumbprint for later.
+  Obtain Your Public IP DNS name:
+   After the service is created, obtain the "Public IP DNS name" from the Azure portal. This URL will serve as your DNS name and Common Name (CN) for further configurations (e.g. bot.contoso.com).
+   ![Public IP DNS name](Images/PublicIPDNSName.png).
 
-3. Set up cloud service configuration
+2. Publish a Cloud Services (classic) project to Cloud Services (extended support) by using Visual Studio:
```
Comment (samarthasthana) [inline Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md:178]: I am confused on the numbering here. The last numbering used before this is also 2

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md [hunk 10]
```diff
@@ -57,25 +59,138 @@ To verify your policy was assigned correctly:
...
+   Match Public IP Name:
+   * Ensure that the name attribute under <PublicIP> (MyPublicIP in this example) matches the name used in your application code to fetch the public IP address dynamically.
+   * To ensure that the domainNameLabel matches between your configuration (.cscfg file) and the Azure portal settings.
+   
+Step 4: Create Your Cloud Service (Extended Support)
+      You can deploy your Azure extended service using various methods, such as the Azure portal, PowerShell, Azure Resource Manager templates, or Visual Studio.
+
+1. To create a Cloud Service (Extended Support) in Azure, follow these steps:
+   
+### Note:
```
Comment (samarthasthana) [inline Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md:154]: What is the note here, its a little unclear with the formatting

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md [hunk 11]
```diff
@@ -57,25 +59,138 @@ To verify your policy was assigned correctly:
...
+
+   Option 1: Upload to Azure Storage Account:
+    * Upload your packaged application (cspkg file) along with the .cscfg and .csdef files to an Azure Storage Account container.
+    * This method allows you to deploy directly from the Azure Storage Account during service creation.
+
+   Option 2: Use Local Files:
+    * Alternatively, you can deploy directly from local files during the service creation process.
+    * Ensure all required files, including .cscfg and .csdef, are accessible and correctly referenced during deployment.
+
+  Create Cloud Service (extended support):
```
Comment (samarthasthana) [inline Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md:171]: Should this be a numbered item?

**Review by samarthasthana — APPROVED** [review]

**Pull request merged**